ColorPatternsTests/ColorPatternsTests.swift

```diff
@@ -7,9 +7,12 @@
 //
 
 import XCTest
+@testable import ColorPatterns
 
 class ColorPatternsTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/101e28ff4ee975d149f242dd211a235cc564fa4a/ColorPatternsTests/ColorPatternsTests.swift#L12"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>